tests/z80benchtests/CLITests.cs

```diff
@@ -12,20 +14,70 @@ public class CLITests {
             "--max-cycles <count>",
             "--vram-compare filename[@offset]]"
         };
+        private StringWriter _stdOut;
+        private StringWriter _stdErr;
+
+        [TestInitialize()]
+        public void Setup() {
+            _stdOut = new StringWriter();
+            _stdErr = new StringWriter();
+            Console.SetOut(_stdOut);
+            Console.SetError(_stdErr);
+            typeof(Z80Bench).TypeInitializer.Invoke(null, null);
```

Comment:
Resets the exposed Func<> seams before every test in case the last one overrode one